src/main/resources/xml/schema/2020-1/nordic2020-1.sch

```diff
@@ -518,7 +518,7 @@
         <rule context="html:*[tokenize(@epub:type, '\s+') = 'footnote']">
             <let name="context" value="concat('(&lt;', name(), string-join(for $a in (@*) return concat(' ', $a/name(), '=&quot;', $a, '&quot;'), ''), '&gt;)')"/>
             <assert test="(parent::html:section)">[nordic204a] 'footnote' must have a parent section. <value-of select="$context"/></assert>
-            <assert test="not(following-sibling::html:*[not(local-name()='aside' and @epub-type='footnote')])">[nordic204a] 'footnote' must be placed at the end of a section.</assert>
+            <assert test="not(following-sibling::html:*[not(local-name()='aside' and @epub:type='footnote')])">[nordic204a] 'footnote' must be placed at the end of a section.</assert>

```

Comment:
`@epub:type = 'footnote'` should be `tokenize(@epub:type, '\s+') = 'footnote'` so that it handles cases with multiple epub:type values.

Reply:
Great catch, Fixed that issue.